### PR TITLE
[CN-736] Custom TLS certificates support

### PR DESCRIPTION
**Pull request by dzeromski-hazelcast**

### CODE REVIEW

File: docs/modules/ROOT/pages/tls.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+== Configuring Hazelcast TLS
+
+Below are the configuration options for the TLS.
+
+[cols="25%m,75%a"]
+|===
+|Field|Description
+
+|`secretName`
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/tls.adoc:16]: @dzeromski-hazelcast Could you provide a sample secret yaml and one-line `kubectl create secret` command?

File: docs/modules/ROOT/pages/tls.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
+= Configuring TLS
+:description: Hazelcast TLS is a security feature that provides encryption and authentication of data transmitted between Hazelcast nodes, preventing unauthorized access and eavesdropping.
+
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/tls.adoc:3]: Could you provide link for hazelcast main TLS doc?

Comment (hasancelik) [inline docs/modules/ROOT/pages/tls.adoc:3]:
```suggestion
:page-enterprise: true
```
Screenshot 2023-03-31 at 10 13 06

**Review by hasancelik — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -17,6 +17,7 @@
 * xref:native-memory.adoc[Native Memory]
 * xref:jvm-parameters.adoc[Configuring JVM Parameters]
 * xref:advanced-networking.adoc[Advanced Networking]
+* xref:tls.adoc[TLS]
```
Comment (kutluhanmetin) [inline docs/modules/ROOT/nav.adoc:20]:
```suggestion
* xref:tls.adoc[Configuring TLS]
```

**Review by kutluhanmetin — APPROVED** [review]
Just a tiny suggestion, LGTM!

**Pull request merged**